src/ResourceManager/Sql/AzureRM.Sql.psd1

```diff
@@ -191,7 +191,17 @@ CmdletsToExport = 'Get-AzureRmSqlDatabaseTransparentDataEncryption',
                'Set-AzureRmSqlServerDnsAlias',
                'New-AzureRmSqlDatabaseRestorePoint',
                'Remove-AzureRmSqlDatabaseRestorePoint',
-               'Stop-AzureRmSqlElasticPoolActivity'
+               'Stop-AzureRmSqlElasticPoolActivity',
+               'Get-AzureRmSqlManagedInstance',
+               'New-AzureRmSqlManagedInstance',
+               'Remove-AzureRmSqlManagedInstance',
+               'Set-AzureRmSqlManagedInstance',
+               'Update-AzureRmSqlManagedInstance',
```

Comment:
Why is there both Set- and Update- ? In SQL we only have Set- commands.

Reply:
Set = PUT (replace), Update = Patch.  This is athe standard for all new cmdlets across the modules

Reply:
Yes, added SET for PUT and Update for Patch